recipes/dcase2021_task4_baseline/finetune_on_separated.py

```diff
@@ -0,0 +1,364 @@
+import argparse
+from copy import deepcopy
+import numpy as np
+import os
+import pandas as pd
+import random
+import torch
+import yaml
+
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks import EarlyStopping, ModelCheckpoint
+from pytorch_lightning.loggers import TensorBoardLogger
+
+from desed_task.dataio import ConcatDatasetBatchSampler
+from desed_task.dataio.datasets import StronglyAnnotatedSet, UnlabelledSet, WeakSet
+from desed_task.nnet.CRNN import CRNN
+from desed_task.utils.encoder import ManyHotEncoder
+from desed_task.utils.schedulers import ExponentialWarmup
+
+from local.classes_dict import classes_labels
+from local.sepsed_trainer import SEPSEDTask4_2021
+from local.sed_trainer import SEDTask4_2021
+
+
+class EnsembleModel(torch.nn.Module):
```

Comment:
Could you add a little comment about what is this class please ?
What I understand is that it is a class to have the predictions for each source, and we aggregate them here ?